### DISCUSSION TITLE
Old Style Layer height display

### DISCUSSION THREAD

**Discussion by JNSmith1717** [UnchartedBull/OctoDash#2785] [Ideas]
Is there a way to get the old style layer height where it showed in mm and not have to use the DisplayLayerProgress plugin.  This plugin is causing my nozzle to rub the build plate and will not adjust for the z offset set in Klipper.  Everything worked fine till i installed the plugin, now i have uninstalled it and just have 0 of __ on screen, i like the older setup with the mm showing better anyways

**Comment by jneilliii** [comment]
I think you just have to uninstall the plugin and disable that setting in octodash's settings and then restart.
